File: src/main/java/org/jdc/template/domain/main/household/Household.java
/*
 * Household.java
 *
 * Created: 10/14/2013 12:14:38
 */



package org.jdc.template.domain.main.household;

import android.database.Cursor;
import android.content.ContentValues;


public class Household extends HouseholdBaseRecord {


    public Household(Cursor cursor) {
        setContent(cursor);
    }

    public Household(ContentValues values) {
        setContent(values);
    }

    public Household() {
    }


}

File: src/main/kotlin/org/jdc/template/domain/main/household/Household.kt
/*
 * Household.kt
 *
 * Created: 02/14/2016 07:02:10
 */



package org.jdc.template.domain.main.household

import android.database.Cursor
import android.content.ContentValues


class Household : HouseholdBaseRecord {


    constructor() {
    }

    constructor(cursor: Cursor) {
        setContent(cursor)
    }

    constructor(values: ContentValues) {
        setContent(values)
    }


}